ODIN_II/SRC/include/odin_types.h

```diff
@@ -535,6 +540,17 @@ struct nnet_t {
                                 //////////////////////
 };
 
+// Enumerator showing on which kinds of hard blocks
+// mixing of soft and hard logic implementation should
+// be performed. The Count should be the
+// the last element. The order of the elements
+// in this enum also determines order of optimizations
+// ADDERS SHOULD BE AFTER MULTS
+enum mix_hard_blocks {
```

Comment:
Avoid making the enum and use the one we already have if possible

Reply:
@jeanlego what's the enum you are talking about?

Reply:
Each block could possibly have an optimization pass.
(Adder, sub, memory, shift, etc... Wich are all block.)

I'm okay losing a bit of space to reduce clutter